src/components/ContentPost/ContentPost.tsx

```diff
@@ -1,58 +1,49 @@
 import { IPublication } from "@/utils/interfaces";
-import {
-  ContentWrapper,
-  Link,
-  Thumb,
-  Title,
-  TitleWrapper,
-  ArrowIcon,
-  DateWrapper,
-  Header,
-  Type,
-  ThumbContainer,
-  DescriptionContainer,
-  Description,
-} from "./ContentPost.styles";
 import { POST_TYPES } from "@/utils/constants";
+import Link from "next/link";
+import ArrowForwardIosIcon from "@mui/icons-material/ArrowForwardIos";
 
 const ContentPost = ({ content }: { content: { fields: IPublication } }) => {
   const { title, thumb, link, date, type, description } = content.fields;
   const dateObj = date ? new Date(date) : null;
   const formattedDate = dateObj ? dateObj.toLocaleDateString("pt-BR") : "";
 
   return (
-    <ContentWrapper>
-      <Header>
-        <Type>
-          {
-            POST_TYPES[
-              type as "additional-content" | "data-panel" | "newsletter"
-            ]
-          }
-        </Type>
-        <DateWrapper>{formattedDate}</DateWrapper>
-      </Header>
-      <Link href={link}>
-        <div>
-          <ThumbContainer>
-            <Thumb
-              src={`https:${thumb.fields.file.url}`}
-              alt=""
-              width={600}
-              height={300}
-            />
-            <DescriptionContainer>
-              <Description>{description || "Acessar postagem"}</Description>
-            </DescriptionContainer>
-          </ThumbContainer>
+    <Link href={link} className="block">
+      <div className="rounded-t-md overflow-hidden m-0 border border-gray-200 shadow-sm">
+        <div className="relative group">
+          <img
+            alt=""
+            src={`https:${thumb.fields.file.url}`}
+            className="h-[225px] w-full block object-cover transition-opacity duration-300 group-hover:opacity-75"
```

Comment:
Maybe 180px would be better

```suggestion
            className="h-[180px] w-full block object-cover transition-opacity duration-300 group-hover:opacity-75"
```